spec/models/concerns/nested_calculator_validation_spec.rb

```diff
@@ -27,7 +27,9 @@
     end
 
     it "adds custom error messages to base" do
-      expect(parent.errors[:base]).to include(/^Amount: Invalid input/)
+      expect(parent.errors[:base]).to include(
+        /#{I18n.t('activerecord.errors.messages.calculator_preferred_value_error')}/
+      )
```

Comment:
I think that we should skip this.

Reply:
Do you mean skip this example group, or just skip this change (and keep the explicit string as you mentioned in the earlier comment).

Reply:
Skip the change. Use explicit strings.